lib/rails_admin/config/fields/types/bson_object_id.rb

```diff
@@ -1,4 +1,9 @@
 require 'rails_admin/config/fields/types/string'
+begin
+  require 'mongoid'
+rescue LoadError => e
+  puts "[RailsAdmin] #{e.message}"
+end
```

Comment:
This message is useless for non-Mongoid users.
Can't we just go without requiring mongoid here?